### ISSUE TITLE
Changed settings for `captionStyleTemplateDefault` take no effect?

### ISSUE THREAD

**Issue by Niklas-Peter** [pospiech/latex#1]
If I change the settings for `captionStyleTemplateDefault` in https://github.com/pospiech/latex/blob/master/latexthesistemplate/template/preamble/style-caption.tex#L3 I can not see any difference in the result.

For testing I downloaded your template again and only changed set `skip = 3cm`. Then I compiled the LaTeXTemplate.tex and on page 9 in the resulting PDF the spacing of the images' captions is still as before. However for subfigures it works if you edit the options in https://github.com/pospiech/latex/blob/master/latexthesistemplate/template/preamble/style-caption.tex#L89 and below.

**Comment by pospiech** [comment]
This is correct. Please take a look at the caption manual. In section 2.6 it says that the skip option is overwritten by many other packages. All of those are used in the template.

That means that I can only classify this as a know bug with no fix.

Matthias

**Comment by Niklas-Peter** [comment]
Thanks.

Is there a quick hack for me to overwrite the setting, e.g. by inserting code directly before `begin{document}` or so? I tried setting `abovecaptionskip` without success.

**Comment by pospiech** [comment]
Sorry, I do not have enough knowledge about the internals of this. I recommend to create a short demo (without my template) and post this as a question on tex.stackexchange.com.
